bitnami/kafka/templates/NOTES.txt

```diff
@@ -72,36 +72,39 @@ You need to configure your Kafka client to access using SASL authentication. To
 
     - kafka_jaas.conf:
 
-        KafkaClient {
-            {{- if .Values.auth.saslMechanisms | regexFind "scram" }}
-            org.apache.kafka.common.security.scram.ScramLoginModule required
-            {{- else }}
-            org.apache.kafka.common.security.plain.PlainLoginModule required
-            {{- end }}
-            username="{{ index .Values.auth.jaas.clientUsers 0 }}"
-            password="$(kubectl get secret {{ $fullname }}-jaas -n {{ $releaseNamespace }} -o jsonpath='{.data.client-passwords}' | base64 --decode | cut -d , -f 1)";
-        };
+cat > kafka_jaas.conf <<EOF
```

Comment:
My concern is that this may not be compatible with people that are running Windows or similar environments. Isn't it easier for users just to instruct the contents of the files (without indentation) so they create it on their own side?

Reply:
Yes, but that was the problem in #3499, that it is very easy to miss the `;` at the end password line .

Reply:
There are also many charts where it is indicated `export ENV_VAR=$(kubectl ....` and that is not valid in windows environments.